src/providers/fault-count/cat-a-mod2/__tests__/fault-count.cat-a-mod2.spec.ts

```diff
@@ -0,0 +1,23 @@
+// import { TestBed } from '@angular/core/testing';
+//
+// import { configureTestSuite } from 'ng-bullet';
+// // import { vehicleChecksTwoFaults, vehicleChecksFiveFaults } from '../../__mocks__/cat-C-test-data-state-object';
+// import { FaultCountAM2Helper } from '../fault-count.cat-a-mod2';
+//
+// fdescribe('FaultCountAM2Helper', () => {
+//
+//   configureTestSuite(() => {
+//     TestBed.configureTestingModule({
+//     });
+//   });
+//
+//   describe('getSafetyAndBalanceFaultCountCatAM2', () => {
+//     it('2 driving faults result in 1 driving faults and 1 serious', () => {
+//       expect((FaultCountAM2Helper as any)
+//         .getSafetyAndBalanceFaultCountCatAM2(vehicleChecksTwoFaults).drivingFaults).toEqual(1);
+//       expect((FaultCountAM2Helper as any)
+//         .getSafetyAndBalanceFaultCountCatAM2(vehicleChecksTwoFaults).seriousFaults).toEqual(1);
+//     });
+//   });
+//
+// });
```

Comment:
Commented out tests? Will these be addressed in a later story - in which case I'd like to see a TODO with story number against it.

Reply:
Hi Ross - I'm addressing that at the mo.  My stories blocking another at the mo so wanted eyes on it whilst im finishing up

Reply:
added todo's